kura/org.eclipse.kura.web2/pom.xml

```diff
@@ -252,7 +252,7 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.3</version>
 				<configuration>
-					<source>1.6</source>
+					<source>1.7</source>
```

Comment:
According to this [document](http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javac.html) the -target is not so relevant if -source is specified